src/sortStudents.ts

```diff
@@ -1,16 +1,56 @@
 
 export interface Student {
-  // describe Student interface
+  name: string,
+  surname: string,
+  age: number,
+  married: boolean,
+  grades: number[],
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name = 'name',
+  Surname = 'surname',
+  Age = 'age',
+  Married = 'married',
+  AverageGrade = 'grades',
 }
 
-// create SortOrder type
-export type SortOrder;
+export type SortOrder = 'asc' | 'desc';
 
+function average(numbers: number[]): number {
+  return numbers.reduce((acc, num) => (acc + num)) / numbers.length;
+}
+
+export function sortStudents(
+  students: Student[],
+  sortBy: SortType,
+  order: SortOrder,
+): Student[] {
+  const studentsCopy = [...students];
+
+  studentsCopy.sort((current: Student, next: Student) => {
```

Comment:
```suggestion
  return [...students].sort((current: Student, next: Student) => {
```
You can immediately return sorted copy.